src/main/java/ch/njol/skript/effects/EffChange.java

```diff
@@ -1,18 +1,18 @@
 /**
- *   This file is part of Skript.

```

Comment:
This PR has **A LOT** of unnecessary line changes. 
A PR should only include changes which are relevant to the idea behind the PR itself.
Fixing spaces in javadocs/copyrights, and also fixes other spacing is not relevant to this PR itself.